drivers/platform/x86/asus-wmi.c

```diff
@@ -1745,6 +1746,14 @@ static void asus_wmi_notify(u32 value, void *context)
 		}
 	}
 
+	if (code == NOTIFY_KBD_BRTTOGGLE) {
+		if (asus->kbd_led_wk < asus->kbd_led.max_brightness)
+			code = NOTIFY_KBD_BRTUP;
+		else {
```

Comment:
For the coding style, the curly brackets are not needed, if the if-else statement has only one line code.